article/Experiment/GCF-ablation-study.tex

```diff
@@ -0,0 +1,32 @@
+\subsection{Ablation study for GCF}
+\textcolor{red}{For GCF we have used the wrong adjacency matrix, that do not contain self connections. We are unsure what the difference will be in the result, but the results might change.}
+Meng Liu et. al does not present an ablation study for BiTGCF and GCF, which LightGCN showed is important with their regards to NGCF \cite{lightgcn,BiTGCF}.
```

Comment:
burde have en mere detaljeret intro om hvad det er vi skal teste i den her ablation study
nævn hvad lightGCN og GCF bruger og hvad vi vil prøve og fjerne

Reply:
ville måske give mening at have equations med igen for at vise hvad det er de gør og hvordan det ser ud når vi fjerne ting